etc/scripts/openhub_scraper.py

```diff
@@ -0,0 +1,28 @@
+# A script to extract openhub license names, their urls, and save it in a json file
+import json
+import urllib2
+import requests
+from bs4 import BeautifulSoup
+
+total_pages = 17
+license_dict = {}
+license_list = []
+
+try:
```

Comment:
Why wrap things in a try/except since you are only printing the error? Also it would be best if you do a proper function for this called in a `if __name__ == '__main__':` block

Reply:
@pombredanne I've added the exception `e` too for the stack trace. Should I keep the `try...except` block or remove it ?